.vscodeignore

```diff
@@ -11,6 +11,7 @@ examples/**
 images/*.psd
 images/*.ai
 images/screenshot_folders.png
```

Comment:
Not really. But as I wasn't sure if the `gif` was going to work I didn't remove it, in case I had to prepare an static image. This is not a finished PR. I wanted you to check the `gif`. If we solve `gif` issues then I'll remove it.